NGitLab/Models/IssueCreate.cs

```diff
@@ -27,5 +27,8 @@ public class IssueCreate
 
         [JsonPropertyName("confidential")]
         public bool Confidential;
+
+        [JsonPropertyName("due_date")]
+        public string DueDate;
```

Comment:
Should be `DateTime`. You may need a custom json converter (`DateOnlyJsonConverter`) like https://github.com/ubisoft/NGitLab/blob/866ec15e8a98eecab7d6b5c892943b42a5509add/NGitLab/Impl/Json/DateTimeConverter.cs#L21 to serialize using `yyyy-MM-dd` format. Before implementing the converter, you should validate if the GitLab API accepts the ISO format.

Reply:
The ISO format didn't seem to work, so I've added a new `DateOnlyConverter` which seems to behave as expected.